packages/react/src/generators/application/application.ts

```diff
@@ -145,7 +145,7 @@ export async function applicationGenerator(
     tasks.push(webpackInitTask);
   } else if (options.bundler === 'rspack') {
     const { configurationGenerator } = ensurePackage(
-      '@nrwl/rspack',
+      '@nx/rspack',
```

Comment:
this is the only file I found referencing the `@nrwl` scoped nx labs packages that wasn't docs based.